### PR TITLE
Update review.md

### PR DESCRIPTION
**Pull request by paulosacramento**
Added a link to the next page, as @Bosch-0 pointed out here: https://github.com/BitcoinDesign/Guide/issues/386#issuecomment-857463411

### CODE REVIEW

**Review by johnsBeharry — APPROVED** [review]
LGTM

**Comment by johnsBeharry** [comment]
Question though, is the "Contribute to guide" section suppose read like the other chapters that require you to move to the next page?

**Comment by GBKS** [comment]
@johnsBeharry I think so.

File: guide/contribute/review.md [hunk 1]
```diff
@@ -112,3 +112,7 @@ Once you visited the URL, locate the page(s) to which proposed changes refer and
 %}
 
 [^1]:[About pull requests](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
+
+---
+
+Next, let's take a look at the [Content guidelines]({{ '/guide/contribute/content-guidelines/' | relative_url }}).
```
Comment (GBKS) [inline guide/contribute/review.md:118]:
```suggestion
Next, let's take a look at the [content guidelines]({{ '/guide/contribute/content-guidelines/' | relative_url }}) that can help inform the review process.
```
How about this addition? It links the next page directly to what you just read on the current page.

**Review by pavlenex — APPROVED** [review]

**Pull request merged**